paseos/attitude/geometric_model.py

```diff
@@ -0,0 +1,123 @@
+from loguru import logger
+import numpy as np
+import trimesh
+
+
+class CuboidGeometricModel:
+    """This model describes the geometry of the spacecraft
+    Currently it assumes the spacecraft to be a cuboid shape, with width, length and height
+    """
+
+    _actor = None
+    _actor_mass = None
+    _actor_height = None
+    _actor_length = None
+    _actor_width = None
+
+    def __init__(
+        self, local_actor, actor_mass, actor_height, actor_length, actor_width
+    ) -> None:
+        """Describes the geometry of the spacecraft, and outputs relevant parameters related to the spacecraft body.
+        Width is the size on the x-direction, Length in the y-direction, and height in the z-direction.
+
+        Args:
+            actor (SpacecraftActor): Actor to model.
+            actor_mass (float): Actor's mass in kg.
+            actor_height: Actor's size in the z-direction.
+            actor_length: Actor's size in the y-direction.
+            actor_width: Actor's size in the x-direction.
+        """
+        logger.trace("Initializing cuboid geometrical model.")
+
+        self._actor = local_actor
+        self._actor_mass = actor_mass
+        self._actor_height = actor_height
+        self._actor_length = actor_length
+        self._actor_width = actor_width
+
+    @property
+    def _find_moi(self):
+        """Actor moment of inertia, currently only suitable for a rectangular prism
```

Comment:
Okay, seeing as part of what you are building may not work for arbitrary meshes, maybe we it would be better to only work with cuboids for now? What do you think @GabrieleMeoni ?

If so, then please disregard the comments regarding meshes and just work with only height,length,width everywhere. @GabrieleMeoni has to decide :)

Reply:
If we work with trimesh, this would work for all meshes. What I was doing there was calculating it by hand, but I've found that probably isn't the most robust method